src/v2/Components/NavBar/Menus/DropDownMenu/DropDownSection.tsx

```diff
@@ -14,7 +14,7 @@ export const DropDownSection: React.FC<DropDownSectionProps> = ({
 
   return (
     <>
-      <Box width={[110, 135, 135, 170, 170]} py={4} mr={[1, 1, 2, 2]}>
+      <Box width="100%" py={4} mr={[1, 1, 2, 2]}>
```

Comment:
And this is the only other real change, which extends the container to full width of the page.